src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+
+  if (days < 3) {
+    result = days * 40;
```

Comment:
```suggestion
    return days * 40;
```
the `result` variable is redundant here - you may immediately return what you need

Also, move "magic numbers" (40, 20, 50) to variables with meaningful names